wd_helper.au3

```diff
@@ -1212,7 +1216,7 @@ Func _WD_SelectFiles($sSession, $sStrategy, $sSelector, $sFilename)
 		EndIf
 	EndIf
 
-	Local $sMessage = $sResult & " file(s) selected"
+	Local $sMessage = $sResult & (($iErr) ? ("") : (" file(s) selected"))
```

Comment:
Have you tested this? AFAICS, it will simply display "0" when an error occurs. I would think it was better before the change or alternatively like this --

`	Local $sMessage = (($iErr) ? ("") : ($sResult & " file(s) selected"))`

Reply:
[_WD_SelectFiles $sMessage](https://github.com/Danp2/au3WebDriver/pull/290/commits/896748b3019dde3c767b76f9c8120708eb835848)